pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -341,40 +349,34 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 			cidrStrings = append(cidrStrings, ipv6Addr.String())
 		}
 	} else {
-		// multi-networking enabled clusters
-		cidrStrings, err = ca.performMultiNetworkCIDRAllocation(node, instance.NetworkInterfaces)
-		if err != nil {
-			nodeutil.RecordNodeStatusChange(ca.recorder, node, "CIDRNotAvailable")
-			return fmt.Errorf("failed to get cidr(s) from provider: %v", err)
+		if hasNodeLabels {
+			cidrStrings = ca.getDefaultNwCIDRs(node, instance.NetworkInterfaces, defaultSubnet, defaultPodRange)
 		}
```

Comment:
It is better to have the code under the "if !multiNetworkingDisabled" check on line 365 here. In fact, we can keep the call that you deleted in this else condition as is.

Reply:
I updated the code after syncing offline, please take a look.